pkgs/applications/misc/vul/default.nix

```
@@ -0,0 +1,31 @@
+{ lib, pkgs, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "vul";
+  version = "unstable-2020-02-15";
+
+  src = fetchFromGitHub {
+    owner = "LukeSmithxyz";
+    repo = pname;
+    rev = "f6ebd8f6b6fb8a111e7b59470d6748fcbe71c559";
+    sha256 = "aUl4f82sGOWkEvTDNILDszj5hztDRvYpHVovFl4oOCc=";
+  };
+
+  makeFlags = [
+    "PREFIX=${placeholder "out"}"
+  ];
+
+  doCheck = true;
+
+  # Tell Shellcheck to ignore a specific redundancy error
+  SHELLCHECK_OPTS = "-e SC2181";
+  checkInputs = [ pkgs.shellcheck ];
```

```suggestion
  checkInputs = [ shellcheck ];
```

Does it do anything else except running shellcheck? If not than it can also be removed because we are not testing the integrity and just doing some linting.